rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/cpp/Cpp17Generator.kt

```diff
@@ -651,32 +652,43 @@ open class Cpp17Generator(flowTransform: FlowTransform,
 
     private fun File.cmakeLists(targetName: String, fileNames: List<String>, toplevelsDependencies: List<Toplevel> = emptyList(), subdirectories: List<String> = emptyList()) {
         mkdirs()
+        if (usingPrecompiledHeaders && !generatePrecompiledHeaders) {
```

Comment:
But why do we need to generate pch and don't use them?

Reply:
I didn't change the way this option is used nowadays.
`usingPrecompiledHeaders` option describes if pch are enabled by default. But if this generated library is used as dependent project - outer project can override this value, but pch files should be generated and present in project anyway.
But we do not use cmake in ubt and we need extra option to not generate pch at all, that's why I added new option.